tests/System/transformationSystem.sh

```diff
@@ -42,10 +42,11 @@ if [ -d "TransformationSystemTest" ]; then
 fi
 echo "Creating TransformationSystemTest"
 mkdir -p TransformationSystemTest
-directory=/lhcb/certification/Test/INIT/$version/$tdate/$stime
+directory=/dteam/DIRAC/certification/Test/INIT/$version/$tdate/$stime
```

Comment:
Is this location OK?

Reply:
Could we maybe agree on a common base location that we would use everywhere ? 
For the `srm` storage, I defined `dteam/diracCertification/` as a base path already, so then it's clean. But for the `RAL` storage, since it's a bucket with no structure, everything is mixed up. So maybe your LFNs could also all start with `/dteam/diracCertification` ? It makes it easier to cleanup after us

Reply:
works for me